src/assets/sass/website/containers/about.scss

```diff
@@ -0,0 +1,118 @@
+@import "../color-palette.scss";
+@import "../variables.scss";
+@import "../mixins.scss";
+@import "../shared/half-clipped-shape.scss";
+
+.about-wrapper {
+  background-color: $primary;
+  height: max-content;
+  margin-top: -$header-height;
+  min-height: calc(100vh - #{$footer-height});
+  @include is-mobile {
+    margin-top: -$header-height-mobile - 45;
+    min-height: calc(100vh - #{$footer-height-mobile});
+  }
+  @media (max-width:600px) {
+    margin-top: -75px;
+  }
+  .hero-wrapper{
+    @media (max-width:600px) {
+      padding-top: 150px;
+    }
+  }
+  .about-container{
+    .half-filled-box{
+      margin: 0!important;
```

Comment:
apply margin without `important` for this line and the upcoming lines :point_down: